source/includes/stackpath/_firewallrules.md

```diff
@@ -0,0 +1,46 @@
+## Firewall Rules
+
+### Create a firewall rule
+
+Restrict access to a site using allow and block rules.
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/firewallrules?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Required Query Params | &nbsp;
+---- | -----------
+`siteId`<br/>*UUID* | The ID of the site for which to create the firewall rule. This parameter is required.
+
+Required Body Attribute | &nbsp;
+------------------------| -----------
+`action`<br/>*string* | Either ALLOW or BLOCK.
+`name`<br/>*string* | The name of the rule.
+`ipStart`<br/>*string* | The start ip adress for the rule. When no `ipEnd` attribute is provided, the rule only applies for the ip provided in `ipStart`.
+
+Optional Body Attribute | &nbsp;
+----------------------- | -----------
+`ipEnd`<br/>*string* | There end ip adress for the rule.
+
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/firewallrules?siteId=f9dea588-d7ab-4f42-b6e6-4b85f273f3db"
+```
+> Request body example for creating a firewall rule:
+
+```js
```

Comment:
I believe this should be `json` instead of `js`, I rebased off your branch and noticed the body is ill formatted 😅
![image](https://user-images.githubusercontent.com/36786091/112527825-d63fd980-8d79-11eb-86ff-974510c3dccd.png)

Reply:
Thanks @aaronalmeida . I fixed it in d3bbdcdc08e640548e1a7e576b84bc9044dea57c